packages/uhk-web/src/app/components/macro/action-editor/tab/mouse/macro-mouse.component.ts

```diff
@@ -120,4 +126,18 @@ export class MacroMouseTabComponent implements OnInit {
         return TabName.Move;
     }
 
+    isMacroValid = () => {
+        switch (this.macroAction.constructor) {
+            case MoveMouseMacroAction:
+            case ScrollMouseMacroAction:
+                const { x, y } = this.macroAction as MoveMouseMacroAction;
+                return x !== undefined && x !== null && y !== undefined && y !== null;
```

Comment:
pls use `return !!x && !!y;`

Reply:
better to keep the longer version as your solution no allow to set `0` to x or y which should be valid. @ert78gb